src/lib/modernExtend.ts

```diff
@@ -1448,7 +1461,17 @@ export function iasZoneAlarm(args: IasArgs): ModernExtend {
     let alarm2Name = 'alarm_2';
 
     if (args.zoneType === 'generic') {
-        args.zoneAttributes.map((attr) => exposes.push(exposeList[attr]));
+        args.zoneAttributes.map((attr) => {
+            if (args.description) {
```

Comment:
I think this can be changed to:

```
let expose = exposeList[attr];
if (args.description) {
  expose = expose.clone().withDescription(args.description);
}
exposes.push(expose);
```

Reply:
ok. try